src/Network/Network/Generated/Models/PSPublicIpAddressDnsSettings.cs

```diff
@@ -32,6 +32,8 @@ namespace Microsoft.Azure.Commands.Network.Models
 {
     public partial class PSPublicIpAddressDnsSettings
     {
+        public PSDomainNameLabelScopeType DomainNameLabelScope { get; set; }
```

Comment:
Previously you added ps1xml attribute on this property. But now, this attribute is removed. Do you need to make it visible from the output object?

Reply:
> Previously you added ps1xml attribute on this property. But now, this attribute is removed. Do you need to make it visible from the output object?

changed